commcare_connect/templates/tailwind/pages/opportunities.html

```diff
@@ -2,13 +2,129 @@
 {% extends 'tailwind/base.html' %}
 
 {% block content %}
+    {% include 'tailwind/components/breadcrumbs.html' with path=path %}
     <div class="container md:max-w-screen-2xl mx-auto flex flex-col gap-4">
-        {% include "tailwind/components/opportunity-dashboard/detials.html" with data=data totalinfo=totalinfo %}
+        <div class="flex flex-col gap-8 w-full bg-brand-deep-purple {% if type == 'test ty vbvbt' %} !text-brand-mango {% endif %} rounded-lg p-4">
+            <div class="flex items-center">
+                <h3 class="text-sm font-medium text-brand-sky flex-1">Program Name</h3>
+                <i class="fa-light fa-bars text-white"></i>
+            </div>
+            <div class="flex justify-between items-end">
+                <div class="flex flex-col gap-2">
+                    <h1 class="text-2xl font-medium text-white block">Opportunity Name</h3>
```

Comment:
This style `text-2xl font-medium text-white` is used above 5 times in the project, shall we have something like `text-header` or some other class that abstracts it out?